GoogleSignIn/Sources/GIDGoogleUser.m

```diff
@@ -40,8 +40,15 @@
 static NSString *const kAudienceParameter = @"audience";
 static NSString *const kOpenIDRealmParameter = @"openid.realm";
 
+// Additional parameter names for EMM.
+static NSString *const kEMMSupportParameterName = @"emm_support";
+
 NS_ASSUME_NONNULL_BEGIN
 
+// A specialized GTMAppAuthFetcherAuthorization subclass with EMM support.
```

Comment:
This should only be built when needed.  Consider organizing it with the rest of the implementation behind `#if TARGET_OS_IOS && !TARGET_OS_MACCATALYST`.  Alternatively, it may be a good time to move these EMM classes out into their own header/implementation files.

Reply:
Are you saying that you've added a new header/implementation files for this class?  I'm not seeing them.